node/src/command.rs

```diff
@@ -53,6 +53,8 @@ fn load_spec(
 		)?)),
 		"dev" | "development" => Ok(Box::new(chain_spec::development_chain_spec(None, None))),
 		"local" => Ok(Box::new(chain_spec::get_chain_spec(para_id))),
+		#[cfg(feature = "test-staking")]
+		"staking" => Ok(Box::new(chain_spec::staking_spec(para_id))),
```

Comment:
This is a great idea. Might make sense to do the same thing for stagenet in the future.